recipes/service-http-connection-pool.rb

```diff
@@ -5,8 +5,6 @@
 # Copyright (C) 2014 Rackspace Hosting
 #
 
-include_recipe 'repose::install'
```

Comment:
It is already called from repose::default.  It is redundant to include it in every filter and service recipe.